app/views/ppd/index.html.haml

```diff
@@ -16,7 +16,7 @@
 
 .row
   %h1.col-md-12
-    Search the price paid dataset:
+    = yield(:title) + ":"
```

Comment:
To prevent rubocop changes down the line, I'd prefer string interpolation rather than concatenation here:

```ruby
= "#{yield(:title)}:"
```